spinta/formats/html/commands.py

```diff
@@ -12,6 +12,8 @@
 
 from itertools import chain
 from itertools import count
+
+from pygments.lexer import inherit
```

Comment:
Panaikink imports, kurie netyčia buvo pridėti (po to gali atsirasti errors, prod/test env, jei to libo versija neatitinka ir bando importinti neegzistuojančias funkcijas).

Reply:
panaikintas